RNTextInputMask.podspec

```diff
@@ -0,0 +1,28 @@
+#
+# Be sure to run `pod lib lint RNTextInputMask.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+    s.name             = 'RNTextInputMask'
+    s.version          = '2.0.0'
```

Comment:
Both of this can be taken from the package.json like this:

```
require "json"

package = JSON.parse(File.read(File.join(__dir__, "package.json")))

s.version = package['version']
```

Reply:
Ok will do that.